pkg/cmd/cli/cmd/rollback.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kubectl "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
+	kcmd "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd"
+
+	describe "github.com/openshift/origin/pkg/cmd/cli/describe"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+)
+
+const rollbackLongDesc = `
+Revert part of an application back to a previous deployment.
+
+When you run this command your deployment configuration will be updated to match
+the provided deployment. By default only the pod and container configuration
+will be changed and scaling or trigger settings will be left as-is. Note that
+environment variables and volumes are included in rollbacks, so if you've
+recently updated security credentials in your environment your previous
+deployment may not have the correct values.
+
+If you would like to review the outcome of the rollback, pass '-o json' to print
```

Comment:
Actually... I think it would be better for you to just implement --dry-run, and have it do the equivalent of -o describe.  Then you can remove the -o support for describe, and it's exactly like get.  You can make '-d' be the shorthand for dry-run (overlaps with describe)